vispy/visuals/volume.py

```diff
@@ -558,6 +626,31 @@ def rescale_data(self):
         self.set_data(self._last_data, clim=self._clim)
         self.update()
 
+    @property
+    def texture_is_inverted(self):
+        if self._texture_limits is not None:
+            return self._texture_limits[1] < self._texture_limits[0]
+
+    def _check_clims_in_texture_range(self, clim):
+        inv_tex_outside_limits = (clim[0] > self._texture_limits[0]) or (clim[1] < self._texture_limits[1])
+        tex_outside_limits = (clim[0] < self._texture_limits[0]) or (clim[1] > self._texture_limits[1])
+        return self.texture_is_inverted and inv_tex_outside_limits or tex_outside_limits
+
+    def _check_clims_in_threshold_range(self, clim):
+        # if the clim range is too small of a percentage of the last-used texture range,
+        # posterization may be visible, so downscale the texture range.
+        range_ratio = np.subtract(*clim) / abs(np.subtract(*self._texture_limits))
+        return np.abs(range_ratio) < self._clim_range_threshold
+
+    def _check_clim_requires_rescale(self, clim):
+        if self._scale_texture_gpu:
+            return False
+        if self._check_clims_in_texture_range(clim):
```

Comment:
elif

Reply:
This seems like a style opinion. I'm not sure I favor one over the other. I think some style checkers don't like `elif` when the previous branch is a `return`.